.github/workflows/ci.yml

```diff
@@ -358,7 +366,15 @@ jobs:
     runs-on: windows-2019
     steps:
     - uses: actions/checkout@v2
+    - uses: actions/cache@v3
+      id: cache-radamsa-build-windows
+      with:
+        # key on the shell script only: this script fixes the 
+        # version to a particular commit, so if it changes we need to rebuild
```

Comment:
We could pass the commit I’d to the script and use it in the key

Reply:
I think it's good for the script to be self-contained